osu.Game/Rulesets/AssemblyRulesetStore.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using osu.Framework.Extensions.ObjectExtensions;
+using osu.Framework.Platform;
+
+#nullable enable
+
+namespace osu.Game.Rulesets
+{
+    /// <summary>
+    /// A ruleset store that populates from loaded assemblies (and optionally, assemblies in a storage).
+    /// </summary>
+    public class AssemblyRulesetStore : RulesetStore
```

Comment:
Maybe `StandaloneRulesetStore` or some other name is better?

Reply:
I think current name is slightly better.